zookeeper-docs/src/main/resources/markdown/zookeeperTools.md

```diff
@@ -425,6 +426,38 @@ All layers compared.
 
 Or use `^c` to exit interactive mode anytime.
 
+
+<a name="zkWatchBenchmark"></a>
+
+### zkWatchBenchmark.sh
+- A benchmark tool that benchmarks the watch throughput and latency, which supports multiple clients threads, multiple watch modes
+- See **ZOOKEEPER-3823** for the design document
```

Comment:
this jira does not contain a design document. I don't think we need to link jira here.